server/utils/courseParticipationUtils.ts

```diff
@@ -83,79 +83,95 @@ export default class CourseParticipationUtils {
   private static summaryListRows(
     courseParticipationWithName: CourseParticipationWithName,
   ): Array<GovukFrontendSummaryListRowWithValue> {
-    const summaryListRows: Array<GovukFrontendSummaryListRowWithValue> = []
+    return [
+      {
+        key: { text: 'Programme name' },
+        value: { text: courseParticipationWithName.name },
+      },
+      CourseParticipationUtils.settingSummaryListRow(courseParticipationWithName),
+      CourseParticipationUtils.outcomeSummaryListRow(courseParticipationWithName),
+      CourseParticipationUtils.outcomeAdditionalDetailSummaryListRow(courseParticipationWithName),
+      CourseParticipationUtils.sourceSummaryListRow(courseParticipationWithName),
+      CourseParticipationUtils.addedBySummaryListRow(courseParticipationWithName),
+    ]
+  }
 
-    summaryListRows.push({
-      key: { text: 'Programme name' },
-      value: { text: courseParticipationWithName.name },
-    })
+  // eslint-disable-next-line @typescript-eslint/member-ordering
+  private static settingSummaryListRow(
+    courseParticipationWithName: CourseParticipationWithName,
```

Comment:
Is it worth making these method parameters a little narrower? E.g. this one could be `setting: CourseParticipationSetting`. It would mean a bit more indexing in the parent method, but less repetitive indexing within the methods

(I'm guessing we're passing around a reference rather than a pointer, so it's less of 'passing smaller things around' issue than a 'doing less indexing' and 'only giving methods what they need' issue)